### PR TITLE
html-forms | ChiaYu Lin

### PR DESCRIPTION
**Pull request by cyulin17**
html-forms

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+<body>
+  <h1>Coding Form!</h1>
+  <form action="example.php" method="get" name="name1">
+    <div>
+      <label for ="address">Email</label><br>
+      <input type="text" name="email" id="address">
+    </div>
+    <div>
+      Why is coding cool?<br>
+      <select name="reason">
```
Comment (BrettAlbright) [inline html-forms/index.html]: Each input requires a label, and you should have matching for and id attributes as per the documentation.

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
Some updates for labels are needed.

File: html-forms/index.html (hunk 1 above)
Comment (cyulin17) [reply html-forms/index.html]: image

**Review by BrettAlbright — APPROVED** [review]
👍

**Pull request merged**